tests/wpt/include.ini

```diff
@@ -92,7 +92,7 @@ skip: true
       [the-script-element]
         skip: false
         [module]
-          skip: true
+          skip: false
```

Comment:
You can simply remove the `[module]` section instead.

Reply:
As I've marked `dynamic-import` and `import-meta` as skipped, I think I will need to remove this `skip: false` 👀?

(Will file follow up issue for both of them when this PR is ready for review 🙏)